### DISCUSSION TITLE
Build iOS Hellow World Failed.we must be register a Apple account that for development？

### DISCUSSION THREAD

**Discussion by hubing53** [AvaloniaUI/Avalonia#9056] [Q&A] [answered]
this error: Xamarin.Shared.targets(1678, 3): Could not find any available provisioning profiles for HelloWorld.iOS on iOS.

**Comment by maxkatz6** [comment, marked as answer]
> we must be register a Apple account that for development

Short answer - yes.
Long answer, follow MS documentation:
https://learn.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/
https://learn.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/manual-provisioning?tabs=windows
https://learn.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/free-provisioning?tabs=windows

**Reply by maxkatz6** [reply]
Or Avalonia documentation, which is basically references to MS one:
https://docs.avaloniaui.net/tutorials/developing-for-mobile/ios/build-and-run-your-application-on-your-iphone-or-ipad
